src/databricks/labs/ucx/queries/assessment/main/08_0_cluster_policies.sql

```diff
@@ -0,0 +1,9 @@
+-- viz type=table, name=Cluster Policies, columns=policy_name,cluster_dbr_version,policy_spark_version
+-- widget title=Cluster Policies, row=8, col=0, size_x=3, size_y=8
+SELECT
+    distinct policy_name,
+    cluster.spark_version as cluster_dbr_version,
+    policy.spark_version as policy_spark_version
+FROM $inventory.clusters as cluster
```

Comment:
why do you need to select from clusters if you are required to select only from policies?

Reply:
From clusters table we will get the cluster DBR versions and from policies table we will get the policy spark version.
We are joining both so that its easy for a customer to compare both side by side and see if a policy spark version can be upgraded based on cluster DBR version